bindata/cronjob-scheduler/daemonset/daemonset.yaml

```diff
@@ -0,0 +1,56 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: cron-scheduler-configmap-ds
```

Comment:
Do we need this file at all? Maybe I don't get it, but... Don't we just modify the deployment for the reconciler?

Reply:
I think I was just copying it because the allowlist had something similar, but I think you're right - the deployment in multus.yaml should be good enough.

Reply:
Let's remove this